scripts/bootstrap-aio.sh

```diff
@@ -53,6 +53,10 @@ else
   export RPCO_APT_ARTIFACTS_AVAILABLE="no"
   rm -f ${BASE_DIR}/group_vars/all/apt.yml
   sed -i '/^# Apt artifact repo configuration$/,$d' ${RPCD_DIR}/etc/openstack_deploy/user_rpco_variables_defaults.yml
+
+  if [[ ${DISTRIB_CODENAME} == "trusty" ]] && ! grep "${DISTRIB_CODENAME}-backports" /etc/apt/sources.list; then
```

Comment:
Why is this being done? The artifacts repo provides the backports.

Reply:
OK, thanks for clarifying via Slack. You may be right with this, but I'll need to step through the whole process to verify that this is not going to cause unwanted side-effects.